### ISSUE TITLE
MCP Eclipse error

### ISSUE THREAD

**Issue by bebeli555** [ModCoderPack/MCPBot-Issues#874]
So i had the MCP 1.12.2 version it was working fine but i went to mess with the minecraft code and it kept crashing on launch so i deleted everything and tried reinstalling the MCP but then there is this error i did everything i did before but now when u run it it says this? how can i fix this pls give a clear help bcs im not very good at this stuff

Error: Exception in thread "main" java.lang.UnsatisfiedLinkError: no lwjgl64 in java.library.path:

**Comment by FinnT730** [comment]
MCP is not a thing anymore. Use Forge to make proper mods.

**Comment by Codetoil** [comment]
I know I don't like it either, but nobody's going to help you jarmod.
